frontend/src/App.js

```diff
@@ -22,6 +22,7 @@ class App extends Component {
     articles: articlesMockData,
     isLoggedIn: true,
     user: this.guestUser,
+    cmsReady: !!localStorage.getItem('allArticles'),
```

Comment:
This seems a bit over the top just as a check, is there not a better way? We load the data somewhere, can we not use that to set cmsReady?

Reply:
daefault false and then set to true when loaded?

Reply:
well yeah. perhaps pass a function down (setCMSReady) to the component that loads the articles?